api/handler/trigger.go

```diff
@@ -79,11 +80,30 @@ func getTrigger(writer http.ResponseWriter, request *http.Request) {
 	if triggerID == "testlog" {
 		panic("Test for multi line logs")
 	}
+
 	trigger, err := controller.GetTrigger(database, triggerID)
 	if err != nil {
 		render.Render(writer, request, err)
 		return
 	}
+
+	if _, templated := request.URL.Query()["templated"]; templated && trigger.Desc != nil {
```

Comment:
You still do query arguments parsing wrong. You need to implement a middleware that parses the query param. Example: https://github.com/moira-alert/moira/blob/master/api/middleware/context.go#L127-L145

Then you need to implement a function that retrieves data from context. Example: https://github.com/moira-alert/moira/blob/master/api/middleware/middleware.go#L74-L77

Then you need to register middleware in router. Example https://github.com/moira-alert/moira/blob/master/api/handler/trigger.go#L32

Remove this code from handler

Reply:
I was saying that `templated` is not a name for query parameter that describes it perfectly. Use for example `render_description` or something like that